examples/quad_android/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "quad_android"
+version = "0.1.0"
+publish = false
+workspace = "../.."
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+
+[features]
+default = ["gl"]
+gl = ["gfx-backend-gl"]
```

Comment:
can we have Vulkan as well?

Reply:
I can try to run Vulkan as well after GL stabilization on CI. `reactivecircus/android-emulator-runner@v2` running an emulator on `MacOS host`. We can't easily run the Android emulator on Linux hosts ([based on README info](https://github.com/ReactiveCircus/android-emulator-runner#can-i-use-this-action-on-linux-vms)).

Reply:
Sounds quite bizarre tbh that an android emulator would run worse on Linux than on macOS. Oh well...